src/lessons/11_dom_part2/dom_part2.slides.html

```diff
@@ -0,0 +1,140 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+  <meta charset="utf-8" />
+  <title>Javascript: Dom manipulations part 2</title>
+  <meta name="date" content="2021-04-26" />
```

Comment:
can we also update the date in here onto 29th?

Reply:
Oh thanks for the hint, I didn't see this.